flexget/plugins/api_tvmaze.py

```diff
@@ -119,6 +203,11 @@ def update(self, series, session):
         self.last_update = datetime.now()
 
         self.genres[:] = get_db_genres(series.genres, session)
+        if series.cast and series.cast.people:
+            self.actors[:], self.characters[:] = get_db_actors_and_characters(series.cast.people, session)
```

Comment:
Will this properly store the link between actor and character? I think maybe we need to make the association table in declarative form, then manually create the association row with all 3 values.